site/_posts/2020-09-22-manageiq-jansa-ga-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "ManageIQ Jansa GA announcement"
+author: Fryguy
+date: 2020-09-22
+tags: releases announcements
+comments: true
+published: true
+---
+TODO: [![ManageIQ Ivanchuk release](/assets/images/blog/Ivanchuk_GA_release.png)](/assets/images/blog/Ivanchuk_GA_release.png)
+
+On behalf of the ManageIQ team, I'm happy to announce the release of **ManageIQ Jansa**!  This release is named after Czech chess grandmaster [Vlastimil_Jansa](https://en.wikipedia.org/wiki/Vlastimil_Jansa).  You can download the Jansa-1 release on our [downloads page](/downloads).
+
+Here are the highlights of the [Jansa release](https://github.com/orgs/ManageIQ/projects/13#column-7337345):
+
+* **Podified Builds** - ManageIQ now has an official release that works on Kubernetes and Red Hat OpenShift.  This is the culmination of the rearchitecture effort that we started during the summer prior to the Hammer release.  In addition to containerization of all of the workers, there is also an orchestrator pod that communicates with Kubernetes and dynanmically spins up or down deployments much in the same way that the appliances server process works.  In this way, we can let Kubernetes do what it does best and manage the pod's lifecycle.  In order to deploy this podified architecture we've also created an operator, which creates the orchestrator as well as the dependent services.  We're very excited to have this new deployment type, and you can expect more enhancements to it in the future.
+
+* **RPM based builds** - The ManageIQ build process has been significantly changed to take advantage of RPMs.  Instead of building the images and appliances from source, now RPMs are built from the source, which are available at https://rpm.manageiq.org. TODO???
```

Comment:
I feel like I need to say more here, but I'm having trouble wording it.  I want to say something about it making builds faster as well as more accurate (or perhaps more predictable? - something about how the RPMs allows the builds to be consistent with each other).  Additionally I might mention the manifests.

Somewhere I also want to mention ubi8 for the images, CentOS 8 for the appliances.  And as such, this is effectively a major version upgrade, requiring existing users to upgrade their appliances.

Reply:
We may expand that and go through the basic building blocks of our stack:
LAMP => LARP = Linux, Apache, Ruby on Rails, PostgreSQL

I know that Rails was upgraded in the Jansa release (https://github.com/ManageIQ/manageiq/issues/20032) and probably Ruby as well.  @simaishi probably knows intimately what OS was used for what build.  Don't think we upgraded Apache and doubtful that we upgraded PG, but I may be wrong.

Reply:
Oh that's an interesting point about
Rails.  I intentionally skipped that one, when I went through the r, but I can make a section on version updates